src/wheel/core/types.h

```diff
@@ -24,3 +24,10 @@ typedef float f32;
 typedef double f64;
 
 extern const usize USIZE_MAX;
+
+#define generic_dynamic_array(type, name)                                              \
```

Comment:
```suggestion
#define VECTOR(type, name)                                              \
```

可以短一些，而且我希望宏尽量用全大写，只有类似函数调用的宏用 snake_case。

Reply:
虽然我之前的代码用的是 list，因为比较短，但现在感觉用 vector 好些，也符合 C++ 的习惯，但现在其他的部分不用改。